libs/components/molecules/src/lib/interactive-svg/interactive-svg.component.ts

```diff
@@ -95,6 +99,18 @@ export class InteractiveSvgComponent implements OnDestroy {
   /** Destroys */
   private destroy$ = new Subject<void>();
 
+  ngOnInit(): void {
+    parse('assets/mapping.csv', {
```

Comment:
Loading and parsing should be done by a service (or possibly in the state) and the passed as an input to the component